### DISCUSSION TITLE
Geomap Panel - Additional Base Layers

### DISCUSSION THREAD

**Discussion by adawson-netafy** [grafana/grafana#41390] [Dashboards, Panels & Visualizations]
I find the provided Geomap base layers sometimes hard to interpret, and would like to add additional base layer maps to the Geomap panel (OSS 8.2.3). 

I use World Topography by Esri on another open-source mapping platform, and it works well. Is it possible to add this in as a layer option?

It would be great to integrate Google Maps via API key and have that be an option for a base layer.

<img width="443" alt="2021-11-06_10-39-28" src="https://user-images.githubusercontent.com/63602127/140613882-5f97fa34-d763-41ae-8c3e-397e417b311f.png">

**Comment by adawson-netafy** [comment]
I solved my own issue - This can be solved via the XYZ tile layer.

Google XYZ Example URL
http://mt0.google.com/vt/lyrs=m&hl=en&x={x}&y={y}&z={z}

**Reply by anilkumar9738** [reply]
![illustration-global-icon_53876-9267](https://github.com/grafana/grafana/assets/114721537/519b4c9e-dfa8-4241-8a73-82536b4d673f)
How can I use this kind of map in grafana (without countries inline border)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
